src/bootstrap/config.rs

```diff
@@ -647,6 +647,17 @@ impl Config {
         config
     }
 
+    /// Try to find the relative path of `bindir`.
```

Comment:
I feel like in ~all cases we'll want to fallback to the absolute path, could we return a &Path and do the fallback here?